.github/workflows/ci.yml

```diff
@@ -32,8 +32,8 @@ jobs:
         with:
           args: --format=txt --diff --dry-run --using-cache=no --verbose
 
-  phpstan:
-    name: PHPStan
+  static-code-analysis:
+    name: Static Code Analysis
```

Comment:
This jobs runs a static code analysis.

It currently runs `phpstan/phpstan` in a step, but it could use an additional or a different tool, for example, `vimeo/psalm`.